src/workspaces/list/RenderedWorkspaces.test.ts

```diff
@@ -39,13 +39,48 @@ describe('The behavior of the RenderedWorkspaces component', () => {
     expect(renderedAzureWS).toHaveLength(1);
   });
 
-  it('should indicate when the workspace is in the process of deleting instead of displaying the description', () => {
+  it.each<{ state: WorkspaceState; statusMessage: string }>([
+    // Cloning and CloningContainer are treated as the same state, for now
+    { state: 'Cloning', statusMessage: 'Workspace cloning in progress' },
+    { state: 'CloningContainer', statusMessage: 'Workspace cloning in progress' },
```

Comment:
I think what I find confusing is that "cloning" means "making a copy of something". The thing you produce is the clone, but the process of cloning involves both the source and the destination, so the message "Workspace cloning in progress" could be equally valid on both the source and the destination.

Reply:
We can also update this later after folks try it out.